src/overlays/actors/ovl_Dm_Ravine/z_dm_ravine.h

```diff
@@ -7,7 +7,10 @@ struct DmRavine;
 
 typedef struct DmRavine {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x8];
+    /* 0x144 */ void (*actionFunc)(Actor*, GlobalContext*); // unused
+    /* 0x148 */ u8 state;
+    /* 0x149 */ u8 loaded;
```

Comment:
I'm not sure if this is the best name but I'm also not entirely sure what this is doing to begin with

Reply:
It's actually redundant against the state. I would recommend the name `isLoaded`